spec/neo4j-core/unit/query_spec.rb

```diff
@@ -639,6 +639,14 @@ def self.it_generates(cypher, params = {})
     describe '.create(q: {Person: {age: nil, height: 70}})' do
       it_generates 'CREATE (q:`Person` {age: {q_Person_age}, height: {q_Person_height}})', q_Person_age: nil, q_Person_height: 70
     end
+
+    describe ".create(q: {:'Child:Person' => {age: 41, height: 70}})" do
+      it_generates 'CREATE (q:`Child`:`Person` {age: {q_Child_Person_age}, height: {q_Child_Person_height}})', q_Child_Person_age: 41, q_Child_Person_height: 70
+    end
+
+    describe ".create(:'Child:Person' => {age: 41, height: 70})" do
+      it_generates 'CREATE (:`Child`:`Person` {age: {Child_Person_age}, height: {Child_Person_height}})', Child_Person_age: 41, Child_Person_height: 70
+    end
```

Comment:
I'm actually trying to fix neo4jrb/neo4j/issues/1133, and I'll definitely defer to you on this (makes sense to me).

I think this means that this line is where my problem arises:  https://github.com/neo4jrb/neo4j/blob/master/lib/neo4j/shared/query_factory.rb#L72 as the .to_sym is combining all the labels into a single sym then passed into here.

...and that the array solution makes it straightforward to fix there too.  Let me know if that all sounds right, thanks Brian